compiler/fir/analysis-tests/testData/resolve/customStringLiterals/classInstanceAsReceiver.kt

```diff
@@ -0,0 +1,13 @@
+class A {
```

Comment:
Fair note.

I've added some test cases for errors in custom string literals conventions. IMO they behave well enough to show user the actual error reason, but they can be improved in some places: e.g don't report `UNRESOLVED_REFERENCE` error and report `MISSING_APPEND_STRING` instead, like it's done in case of `for` loop. 

I'll add better error diagnostics a bit later.